editor/plugins/spatial_editor_plugin.cpp

```diff
@@ -1917,7 +1917,9 @@ void SpatialEditorViewport::_nav_zoom(Ref<InputEventWithModifiers> p_event, cons
 }
 
 void SpatialEditorViewport::_nav_orbit(Ref<InputEventWithModifiers> p_event, const Vector2 &p_relative) {
-
+	
```

Comment:
There is some trailing whitespace on this line and line 1922, could you amend the commit to remove them? See http://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html